imap_processing/hi/l1a/science_direct_event.py

```diff
@@ -266,11 +267,15 @@ def create_dataset(de_data_list: list, packet_met_time: list) -> xr.Dataset:
     # uncomment this once Maxine's PR is merged
     # attr_mgr.add_global_attribute("Data_version", data_version)
 
+    epoch_attrs["CATDESC"] = (
+        "Direct Event time, number of nanoseconds since J2000 with leap "
+        "seconds included"
+    )
```

Comment:
Should this be different? I think it'd be nice to be consistent everywhere and not have this one outlier.

Reply:
I wondered that too but didn't want to assume. @subagonsouth may know more

Reply:
Sorry that I missed this mention. It is probably fine to change. I just wanted to highlight that this epoch variable is storing the direct event times, not packet creation times like many other epoch variables in l1a.